SEOImageUpdates: User Story 1463880, Small PR 2 (#12338)
dotnet/docs: docs/framework/windows-workflow-foundation/how-to-create-a-state-machine-workflow.md

  
     The following example illustrates the completed workflow.  
  

>      ![Completed state machine workflow from the Getting Started Tutorial](./media/how-to-create-a-state-machine-workflow/getting-started-tutorial-complete-state-machine-workflow.jpg)  

Reviewer: ```suggestion
     ![Illustration that shows the completed state machine workflow.](./media/how-to-create-a-state-machine-workflow/complete-state-machine-workflow.jpg)  
```
